### PR TITLE
Update extensionsGallery.json

### PR DESCRIPTION
**Pull request by LycanII**
to add lz-scripts to the extensions gallery.

This PR fixes #

### CODE REVIEW

**Comment by Charles-Gagnon** [comment]
Please : 

1. Add this to [extensionsGallery-insider.json](https://github.com/microsoft/azuredatastudio/blob/release/extensions/extensionsGallery-insider.json) as well
2. Use `https://raw.githubusercontent.com` URLs instead of `https://github.com` for links in the files section of the entry

**Comment by LycanII** [comment]
> Please :
> 
> 1. Add this to [extensionsGallery-insider.json](https://github.com/microsoft/azuredatastudio/blob/release/extensions/extensionsGallery-insider.json) as well
> 2. Use `https://raw.githubusercontent.com` URLs instead of `https://github.com` for links in the files section of the entry

Oh ok noted, i'll adjust right away

File: extensionsGallery-insider.json [hunk 1]
```diff
@@ -4507,6 +4507,67 @@
...
+					},
+					"versions": [
+						{
+							"version": "0.0.1",
+							"lastUpdated": "27/02/2022",
+							"assetUri": "",
+							"fallbackAssetUri": "fallbackAssetUri",
+							"files": [
+								{
+									"assetType": "Microsoft.VisualStudio.Services.VSIXPackage",
```
Comment (Charles-Gagnon) [inline extensionsGallery-insider.json:4529]: For 3rd party extensions we don't support direct links to the VSIX unfortunately - you will need to use the `Microsoft.SQLOps.DownloadPage` assetType and provide a link to the download page (e.g. https://github.com/LycanII/LzScripts/releases/tag/Latest for yours)

Comment (LycanII) [reply extensionsGallery-insider.json]: Done please

**Pull request merged**